modules/file-extension-common/pom.xml

```diff
@@ -64,13 +66,24 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
+                <groupId>org.mule.plugins</groupId>
+                <artifactId>mule-app-plugins-maven-plugin</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+                <extensions>true</extensions>
+            </plugin>
+
+            <plugin>
                 <artifactId>maven-jar-plugin</artifactId>
                 <executions>
                     <execution>
+                        <id>mule-plugin</id>
+                        <phase>package</phase>
                         <goals>
-                            <goal>test-jar</goal>
```

Comment:
I think that test-jar is need because file and ftp use/extend test classes from common